circle.yml

```diff
@@ -3,9 +3,9 @@ machine:
     PROJECT_ROOT: $PWD/unity-buy-sdk
     UNITY_DOWNLOAD_DIR: $PROJECT_ROOT/unity
     UNITY_PKG_LOCATION: $UNITY_DOWNLOAD_DIR/Unity.pkg
-    UNITY_PKG_URL: http://netstorage.unity3d.com/unity/fdbb5133b820/MacEditorInstaller/Unity-5.3.4f1.pkg
+    UNITY_PKG_URL: http://netstorage.unity3d.com/unity/29055738eb78/MacEditorInstaller/Unity-5.3.6f1.pkg
```

Comment:
Just for documentation purposes. Why are we bumping the version that CI is running?